From 17f0b2b710dbf0b72c804323fbd17c4b04b2a2ad Mon Sep 17 00:00:00 2001
From: Takeshi Kihara <takeshi.kihara.df@renesas.com>
Date: Wed, 8 Feb 2017 15:37:38 +0900
Subject: [PATCH 1764/2066] arm64: dts: r8a7795-salvator-xs: Add HS-USB device
 support

This patch adds HS-USB ch0 and ch3 device support for Salvator-X 2nd
version board on R8A7795 ES2.0 SoC.

Signed-off-by: Takeshi Kihara <takeshi.kihara.df@renesas.com>
[zou:Original patch taken from
git://git.kernel.org/pub/scm/linux/kernel/git/horms/renesas-bsp.git
 v4.9/rcar-3.5.1]
Signed-off-by: zou cao <cao.zou@windriver.com>
---
 .../arm64/boot/dts/renesas/r8a7795-salvator-xs.dts |    8 ++++++++
 1 files changed, 8 insertions(+), 0 deletions(-)

diff --git a/arch/arm64/boot/dts/renesas/r8a7795-salvator-xs.dts b/arch/arm64/boot/dts/renesas/r8a7795-salvator-xs.dts
index 51ba52a..68d1ff7 100644
--- a/arch/arm64/boot/dts/renesas/r8a7795-salvator-xs.dts
+++ b/arch/arm64/boot/dts/renesas/r8a7795-salvator-xs.dts
@@ -239,3 +239,11 @@
 &ohci3 {
 	status = "okay";
 };
+
+&hsusb0 {
+	status = "okay";
+};
+
+&hsusb3 {
+	status = "okay";
+};
-- 
1.7.5.4

